test/srtp_driver.c

```diff
@@ -292,9 +322,26 @@ main (int argc, char *argv[])
             printf("failed\n");
             exit(1);
         }
+        printf("testing srtp_protect and srtp_unprotect against "
+               "reference packets with encrypted extensions headers\n");
+        if (srtp_validate_encrypted_extensions_headers() == srtp_err_status_ok)
+            printf("passed\n\n");
+        else {
+            printf("failed\n");
+            exit(1);
+        }
+
+#ifdef OPENSSL
```

Comment:
Really happy to see that you not only added unit test cases, but you also considered the OpenSSL use cases.  If only all code contributors would do this. :-)